File: CCC/modernart.cpp
#include <iostream>

using namespace std;

int main(){
    int m;
    int n;
    int k;
    cin>>m;
    cin>>n;
    cin>>k;
    
    int rows[m];
    int cols[n];

    for (int i = 0; i < k; i++)
    {
        char ch; 
        int idx;
        cin >> ch >> idx;
        idx--;
        if (ch =='R')
        {
            rows[idx] ^=1;
        }else
        {
            cols[idx] ^=1;
        }                
    }
    int ans =0;
    for (int i = 0; i < m; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (rows[i] ^ cols[j])
            {
                ans+=1;
            }
            
        }        
    }

    cout<<ans<<endl;
    
    
}

File: CCC/modernArt.py
m = int(input())
n = int(input())
k = int(input())

rows = [False for _ in range(m)]
cols = [False for _ in range(n)]

for _ in range(k):
    dir,num = input().split()
    num = int(num)
    if dir == 'R':
        rows[num-1] = ~rows[num-1]
    else:
        cols[num-1] = ~cols[num-1]

ans = 0
for i in range(m):
    for j in range(n):
        if rows[i] != cols[j]:
            ans += 1

print(ans)